ZelBack/src/services/appsService.js

```diff
@@ -7856,7 +7876,7 @@ async function appHashHasMessageNotFound(hash) {
  * @param {number} height Block height.
  * @param {number} valueSat Satoshi denomination (100 millionth of 1 Flux).
  * @param {number} i Defaults to value of 0.
- * @returns {void} Return statement is only used here to interrupt the function and nothing is returned.
+ * @returns {boolean} Return true if app message is already present otherwise nothing is returned.
```

Comment:
I do not like much the inconsistency in the return of the function and and that we do not know how long it will take to execute

Reply:
inconsistency fixed.